products/rhel9/profiles/ospp.profile

```diff
@@ -361,20 +361,33 @@ selections:
     - audit_basic_configuration
     - audit_immutable_login_uids
     - audit_create_failed
+    - audit_create_failed_aarch64
     - audit_create_success
+    - audit_create_success_aarch64
     - audit_modify_failed
+    - audit_modify_failed_aarch64
     - audit_modify_success
+    - audit_modify_success_aarch64
     - audit_access_failed
+    - audit_access_failed_aarch64
     - audit_access_success
     - audit_access_success.severity=info
     - audit_access_success.role=unscored
+    - audit_access_success_aarch64
```

Comment:
It looks like audit_access_success_aarch64 should also set its severity to info and role to unscored, similary to audit_rules_success.

Reply:
Thanks for noticing it, fixed.